src/Common/DoctrineHeartbeatMiddleware/SchedulePeriodicHeartbeatMiddleware.php

```diff
@@ -21,10 +21,6 @@ public function __construct(
 
     public function wrap(Driver $driver): Driver
     {
-        if (php_sapi_name() !== 'cli') {
```

Comment:
This got moved, so `Parameters::removeDriverOptions()` is always called, see commit description.